src/modal/body/updateCard/updateCard.tsx

```diff
@@ -0,0 +1,80 @@
+import { useEffect, useState } from 'react';
+import { ModalProps } from '../..';
+import Button from '../../../components/button';
+import Input from '../../../components/input';
+import SelectButton from '../../../components/select-button';
+import './updateCard.css';
+import { useAppDispatch } from '../../../hooks/useAppDispatch';
+import {
+  setRemoveTrelloCard,
+  setUpdateTrelloCard,
+} from '../../../store/reducers/trelloReducer';
+import { useAppSelector } from '../../../hooks/useAppSelector';
+
+const UpdateCard: React.FC<ModalProps> = ({ meta, closeModal }) => {
+  const [value, setValue] = useState('');
+  const dispatch = useAppDispatch();
+  const data = useAppSelector((state) => state.trello.data);
+  const [selectedTitle, setSelectedTitle] = useState(meta.title);
```

Comment:
const dispatch = useAppDispatch(); en üst olmalı.
 
 1. dispatch
 2. useSelector
 3. herhangi useState
 
 örneğini
  const dispatch = useAppDispatch();
  const data = useAppSelector((state) => state.trello.data);
  const [value, setValue] = useState('');
  
  böyle olmalı.

Reply:
bunu da dediğin gibi hallettim